jenkins-pipelines/openshift/Jenkinsfile

```diff
@@ -46,10 +50,10 @@ node ('openstack-jenkins-slave') {
     }
 
     stage('setup HEAT environment') {
-        sh('source ./openstack_rc.sh ; echo -e "parameter_defaults:\n  key_name: jenkins\n  domain_suffix: \$DOMAIN_SUFFIX\n  openshift_openstack_username: \$OS_USERNAME\n  s3_region_endpoint: \$S3_REGION_ENDPOINT\n  s3_access_key: \$S3_ACCESS_KEY\n  s3_secret_key: \$S3_SECRET_KEY\n  s3_bucket_name: \$S3_BUCKET_NAME\n  setupMonitoring: true\n  controlplane_floating_ip: \$CONTROL_PLANE_IP\n  dataplane_floating_ip: \$DATA_PLANE_IP\n  do_upgrades: false\n  worker_small_scale: \$WORKER_SMALL_SCALE\n  worker_medium_scale: \$WORKER_MEDIUM_SCALE\n  worker_large_scale: \$WORKER_LARGE_SCALE\n  openshift_version: \$OCP_VERSION\n  set_node_routes: \$SET_NODE_ROUTES\n  node_routes: \$NODE_ROUTES" > environment.yaml')
+        sh('source ./openstack_rc.sh ; echo -e "parameter_defaults:\n  key_name: jenkins\n  domain_suffix: \$DOMAIN_SUFFIX\n  openshift_openstack_username: \$OS_USERNAME\n  s3_region_endpoint: \$S3_REGION_ENDPOINT\n  s3_access_key: \$S3_ACCESS_KEY\n  s3_secret_key: \$S3_SECRET_KEY\n  s3_bucket_name: \$S3_BUCKET_NAME\n  setupMonitoring: true\n  controlplane_floating_ip: \$CONTROL_PLANE_IP\n  dataplane_floating_ip: \$DATA_PLANE_IP\n  do_upgrades: false\n  worker_small_scale: \$WORKER_SMALL_SCALE\n  worker_medium_scale: \$WORKER_MEDIUM_SCALE\n  worker_large_scale: \$WORKER_LARGE_SCALE\n  openshift_version: \$OCP_VERSION\n  set_node_routes: \$SET_NODE_ROUTES\n  node_routes: \$NODE_ROUTES\n  multinetwork: \$MULTINETWORK\n  deploy_extra_gateway: \$DEPLOYEXTRAGATEWAY" > environment.yaml')
```

Comment:
this line is getting a bit long. I'd be tempted to break it up. e.g. 
source ...
echo "parameter_defaults:" > environment.yaml
echo "  key_name: jenkins >> environment.yaml
etc.

Or maybe some JSON so that the number of spaces isn't important. (JSON is valid YAML)